### DISCUSSION TITLE
Is this a bug? stats.loggingDebug isn't ignored when stats.logging is set to false.

### DISCUSSION THREAD

**Discussion by denik1981** [webpack/webpack#13976] [Q&A]
Based on webpack official documentation on how [stats.loggingDebug](https://webpack.js.org/configuration/stats/#statsloggingdebug) works, when stats.logging is set to false, this option should be ignored. Well, that's not the case based on my latest builds (I have tried with stats.logging:'none' as well).

Just want to open a discussion here to be sure if this is a bug to report or if I'm doing something wrong.

**Comment by alexander-akait** [comment]
Can you provide example of the problem?
